code/modules/admin/verbs/adminpm.dm

```diff
@@ -162,8 +162,8 @@
 		C.ckey_last_pm = ckey*/
 
 	//play the recieving admin the adminhelp sound (if they have them enabled)
-	//non-admins shouldn't be able to disable this
-	if(C.prefs.sound & SOUND_ADMINHELP)
+	//non-admins always hear the sound, as they cannot toggle it
+	if((!C.holder) || (C.prefs.sound & SOUND_ADMINHELP))
```

Comment:
Everyone with an admin holder (admins, mentors, even mods if we still used them) can turn off bwoinks. We *could* check for every right any of them would have, but simply checking for the holder is cleaner.